flank_wrapper/flankw.sh

```diff
@@ -0,0 +1,29 @@
+download_flank() {
+  echo "Downloading Flank $1..."
+  curl -L https://github.com/Flank/flank/releases/download/$1/flank.jar -o $FLANK_FILE --silent
+}
+
+get_remote_revision() {
+  curl -H "Accept: application/vnd.github.v3+json" https://api.github.com/repos/flank/flank/commits/v21.06.1 -s | grep '"sha":' | head -n 1 | awk '{print $2}' | sed s/\",//g | sed s/\"//g
```

Comment:
I think here we should use $REMOTE_VERSION instead of hardcoded v21.06.1 wdyt?

Reply:
As we agreed with standup we will deliver Kotlin based version

Reply:
but any way this is huge mistake from my side :D thanks